src/wallet/wallet.cpp

```diff
@@ -2564,8 +2564,10 @@ bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CWalletT
                         txNew.vout.insert(position, newTxOut);
                     }
                 }
-                else
+                else {
```

Comment:
Weak nit: Would love if this was `} else {` personally (i.e. no newline after `}`).